### PR TITLE
Release 2.1.1

### PR DESCRIPTION
**Pull request by antonio2368**

### CODE REVIEW

**Review by kostasrim — APPROVED** [review]

File: docs/changelog.md [hunk 1]
```diff
@@ -4,14 +4,27 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.1.1 - This year I hope
+
+### Major Features and Improvements
+
+* Added procedures for retrieving configuration information specific for each stream type.
```
Comment (jbajic) [inline docs/changelog.md:11]:
```suggestion
* Added procedures for retrieving specific information for each stream type.
```

**Comment by antonio2368** [comment]
@g-despot take a look and merge :eyes:

**Review by g-despot — APPROVED** [review]

**Comment by g-despot** [comment]
@antonio2368 what will be the release date?

**Comment by antonio2368** [comment]
> @antonio2368 what will be the release date?

Tomorrow (2021-12-08)

**Pull request merged**